assets/js/analyzer.js

```diff
@@ -1,5 +1,6 @@
 var analyzers = {}, analyzerData = {};
 var currentAnalyzerRequest;
+var THRESHOLD_REQUEST_LENGTH_FOR_ANALYZE = 2020;
```

Comment:
This variable should only be in one place. The current state of this code is a bit repetitive. You might want to move a method into `utils.js`